host-interaction/registry/create/set-registry-value.yml

```diff
@@ -2,17 +2,29 @@ rule:
   meta:
     name: set registry value
     namespace: host-interaction/registry/create
-    author: moritz.raabe@fireeye.com
+    author:
+      - moritz.raabe@fireeye.com
+      - michael.hunhoff@fireeye.com
     scope: function
     mbc:
       - Operating System::Registry::Set Registry Key [C0036.001]
     examples:
       - BFB9B5391A13D0AFD787E87AB90F14F5:0x13147AF0
+      - B5F85C26D7AA5A1FB4AF5821B6B5AB9B:0x40433E
+      - B5F85C26D7AA5A1FB4AF5821B6B5AB9B:0x40415E
   features:
     - and:
-      - or:
+      - optional:
         - match: create registry key
         - match: open registry key
       - or:
         - api: advapi32.RegSetValue
         - api: advapi32.RegSetValueEx
+        - api: advapi32.RegSetKeyValue
+        - api: RtlWriteRegistryValue
+        - api: ZwSetValueKey
+        - api: SHSetValue
+        - api: SHRegSetPath
+        - api: SHRegSetValue
+        - api: SHRegSetUSValue
+        - api: SHRegWriteUSValue
```

Comment:
wow these are becoming quite thorough

Reply:
Thanks for adding these in.  Once these are added in #199 can probably be closed!  :+1: